lib/test/utils/decoderTest.ts

```diff
@@ -0,0 +1,21 @@
+import { after, before, describe } from 'node:test';
+
+const TD = TextDecoder;
```

Comment:
(Optional) more meaningful name, e.g. NativeTextDecoder?

```suggestion
const NativeTextDecoder = TextDecoder;
```

Wdyt?